doc/api/inspector.md

```diff
@@ -10,6 +10,22 @@ It can be accessed using:
 const inspector = require('inspector');
 ```
 
+## inspector.open([port[, host[, wait]]])
+
+Activate inspector on host and port. Equivalent to `node
+--inspect=[[host:]port]`, but can be done programatically after node has
+started.
+
+If wait is `true`, will block until a client has connected to the inspect port.
```

Comment:
Maybe write something like `will block and pass the flow control to the debugger client.` or something else to that effect...